app/Run.hs

```diff
@@ -160,7 +155,7 @@ runNode nodeCli@NodeCLIArguments{..} loggingLayer cc = do
 -- | Sets up a simple node, which will run the chain sync protocol and block
 -- fetch protocol, and, if core, will also look at the mempool when trying to
 -- create a new block.
-handleSimpleNode :: forall blk. (RunNode blk, TraceConstraints blk)
+handleSimpleNode :: forall blk. (RunNode blk, TraceConstraints blk, Show (GenTxId blk))
```

Comment:
Why is this needed? If it is needed for tracing purposes, it should be added to the `TraceConstraints` constraint synonym.

Reply:
cc/ @cleverca22